projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/Messages.kt

```diff
@@ -41,7 +43,18 @@ internal fun showMessage(project: Project, header: String, message: String): Boo
 
   if (widget is ProjectorStatusWidget) {
     val gotItMessage = GotItMessage.createMessage(header, message).setDisposable(widget)
-    gotItMessage.show(RelativePoint.getCenterOf(widget.component), Balloon.Position.above)
+
+    with(widget.component) {
+      gotItMessage.show(
+        object : PositionTracker<Balloon?>(this) {
+          override fun recalculateLocation(baloon: Balloon): RelativePoint {
+            return RelativePoint(widget.component, Point(this@with.size.width / 2,
+                                                         this@with.size.height / 2))
```

Comment:
I just spotted this PR. I haven't tested but I'm wondering if this code can be improved. For example:

```suggestion
            return RelativePoint(this@with, Point(this@with.size.width / 2,
                                                         this@with.size.height / 2))
```

Or even (as before):

```suggestion
            return RelativePoint.getCenterOf(this@with)
```

Reply:
Nice catch! I like more your last suggestion. Commit it with one minor change.